### ISSUE TITLE
Door is sending scans in Z time not A time.

### ISSUE THREAD

**Issue by brackendawson** [somakeit/door-controller2#21]
time.time() returns seconds since epoch in local time.

**Comment by benjie** [comment]
"seconds since epoch" doesn't change based on timezone, daylight savings, or other such things - it's the number of seconds since `1970-01-01T00:00:00Z` which is timezone independent.

(Though technically it's not, since it doesn't account for leap seconds and the like - it factors all days as having 24*60*60 seconds. But near enough.)

**Comment by benjie** [comment]
Oh; I see - you mean sending a string rather than a number. Gotya.

**Comment by brackendawson** [comment]
No, I think I was wrong. time.time is reporting in UTC.

Something is causing the members area to show a scan times that coincide with UTC but are apparently localtime, that should be BST.

tl;dr the scans in the members area are an hour behind.

**Comment by benjie** [comment]
The members area is probably displaying them in UTC?

**Comment by benjie** [comment]
Filed against members area. Really need to add an admin role to the members area so I can see the admin panel again!

**Comment by brackendawson** [comment]
Not going to fix this as the code is no longer used.
